src/Form/Form.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import Form from './Form';
```

Comment:
We'll have to check and see if 'Form' works or if we need to change it to the Material-ui component that we're using (i.e., ControlledOpenSelect). Maybe we can refactor our code to make the components match the filenames.